src/pmpo_fortran.f90

```diff
@@ -77,6 +118,27 @@ subroutine polympo_getMPVelArray(mpMesh, n, array) &
     type(c_ptr), value :: array
   end subroutine
   !---------------------------------------------------------------------------
+  !> @brief enable the set mesh. The mesh is decleared uneditable all the time.
+  !>        if changes/settings are needed, call this function first, then call
+  !         set functions
+  !> @param mpMesh(in/out) the MPMesh is valid/created
+  !---------------------------------------------------------------------------
+  subroutine polympo_startMeshFill(mpMesh) &
+             bind(C, NAME='polympo_startMeshFill')
+    use :: iso_c_binding
+    type(c_ptr), value :: mpMesh
+  end subroutine
+  !---------------------------------------------------------------------------
+  !> @brief disable the set mesh. Once the changes/settings are done,
+  !>        call this function to prevent unexpected changes/settings
```

Comment:
```suggestion
  !> @brief Disabled the modification of mesh topology (number of entities and entity adjacencies). 
  !>        Once the topology is set, call this function to prevent unexpected/accidental modifications.
```